python/ray/_private/runtime_env/context.py

```diff
@@ -25,6 +25,8 @@ def __init__(
         resources_dir: Optional[str] = None,
         container: Dict[str, Any] = None,
         java_jars: List[str] = None,
+        cwd: Optional[str] = None,
+        symlink_dirs_to_cwd: List[str] = None,
```

Comment:
Could you please add a comment somewhere defining `symlink_dirs_to_cwd`? From the name it sounds like we symlink each dir to the cwd.  But actually, for each dir we create symlinks for all of its contents.

Reply:
Sure. I will add some comments.